recipes/LibriSpeech/self-supervised-learning/BEST-RQ/hparams/BEST-RQ-streaming.yaml

```diff
@@ -0,0 +1,197 @@
+# ############################################################################
+# Model: Best-RQ
+# Encoder: Conformer Encoder w/Random Projection Quantizer
+# Training: Librispeech 960h
+# Authors:  Ryan Whetten 2024
+#           Jarod Duret  2024
+# ############################################################################
+# Seed needs to be set at top of yaml, before objects with parameters are made
+
+seed: 1000
+__set_seed: !apply:speechbrain.utils.seed_everything [!ref <seed>]
+data_folder: !PLACEHOLDER # /path/to/LibriSpeech/
+run_name: !PLACEHOLDER
+output_folder: !ref results/<seed>/<run_name>
```

Comment:
I think you should rename `run_name` to `experiment_name`. I would also suggest `output_folder` to be `output_folder: !ref results/<experiment_name>/<seed>/` to remain consistent with some other yamls that already introduced this feature